plugin.video.videomediaset/changelog.txt

```diff
@@ -0,0 +1,67 @@
+[B]2.0.0[/B]
```

Comment:
Can you drop this from the submission? It is no longer exposed to the Kodi interface since krypton